src/components/Header.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+function Header() {
+    return(
+        <div>
+            <header className="bg-sky-200 w-screen h-28 flex justify-between">
```

Comment:
instead of sky-200 in tailwind Config set colors and insert all main colors from color palleter for rehub application and use them with bg-<color name from config file>. It will be easier to change those colors there and to keep track of them and use consistent colors over whole frontend.